lib/phonebook/tools/function.dart

```diff
@@ -8,27 +7,20 @@ import 'package:myecl/phonebook/class/membership.dart';
 int getPosition(
   CompleteMember member,
   String associationId,
-  List<String> rolesTags,
 ) {
   Membership membership = member.memberships
       .firstWhere((element) => element.associationId == associationId);
```

Comment:
A bit beyond the scope of this PR but while fixing the ordering it may be good, for now, to check if the membership mandate_year match the association mandate_year

Reply:
Yeah, I noticed this, I changed it in the lasts commits